articles/data-explorer/ingest-data-event-grid.md

```diff
@@ -190,6 +190,16 @@ If you don't plan to use your event grid again, clean up **test-hub-rg**, to avo
 
 1. In the new window, enter the name of the resource group to delete (*test-hub-rg*), and then select **Delete**.
 
+## Optional : Set ingestion properties via blob metadata
+
+You can specify Ingestion properties of the blob ingestion via the blob metadata. Setting the blob metadata is done by [Azure Cloud Shell](https://docs.microsoft.com/en-us/cli/azure/storage/blob/metadata?view=azure-cli-latest#az-storage-blob-metadata-update) or [Azure Storage Client Library](https://docs.microsoft.com/en-us/azure/storage/blobs/storage-properties-metadata#setting-and-retrieving-metadata).
```

Comment:
Since this is a quickstart per @orspod we shouldn't normally add links outside of this page

Reply:
@radennis, we are moving all these topics to How-to section so we have more flexibility on the links and "additional info"

Reply:
shouldn't this show an explicit example?